README.md

```
@@ -0,0 +1,27 @@
+babysitting and the context switching.
+
+## Requirements
+
+- Python 3
+- Nothing else — standard library only
+
+## Usage
+
+Open two terminals. In each one:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)